src/main/java/com/quizapp/enterprise/models/Question.java

```diff
@@ -16,7 +16,7 @@
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @Column(name = "questionid")
+    @Column(name = "question_id")
```

Comment:
For consistency, this should use `snake_case` just like the other foreign key identifiers in the other models